src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -87,46 +87,18 @@ namespace OpenMS
     double mz = -1.;
     double intensity = 0.;
 
+    PeakAnnotation() = default;
+    PeakAnnotation(String annotation, int charge, double mz, double intensity):
```

Comment:
not needed... just write PeakAnnotations("bla", 43, 34.3, 34)

Reply:
It is for emplace_back